CMake/AbseilFindDependencies.cmake

```diff
@@ -0,0 +1,72 @@
+#
+# Copyright 2017 The Abseil Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+include(ExternalProject)
+
+#
+## pthread
+#
+find_package(Threads REQUIRED)
+
+#
+## CCTZ
+#
+find_package(CCTZ)
+if(NOT CCTZ_FOUND)
+    # if not found, try to bundle it
+    message(STATUS "No External CCTZ found, Use bundle CCTZ package")
+
+    set(CCTZ_BUNDLE_INTERNAL_INSTALLDIR "${CMAKE_CURRENT_BINARY_DIR}/CCTZ_BUNDLE-prefix/install")
+
+    ExternalProject_Add(CCTZ_BUNDLE
+        GIT_REPOSITORY "https://github.com/google/cctz.git"
+        GIT_TAG "d5e227e6bfdb365abb12ec7528d79292732ce0bc"
```

Comment:
nit: bazel uses the master branch not a fixed commit/tag

Reply:
Fixed